addon/helpers/did-update.ts

```diff
@@ -23,6 +23,8 @@ export default class DidUpdateHelper extends Helper {
     );
     if (!this.didRun) {
       this.didRun = true;
+      positional.forEach(() => {});
+      Object.values(named).forEach(() => {});
       return;
     }
     fn(positional.slice(1), named);
```

Comment:
```suggestion
    if (!this.didRun) {
      this.didRun = true;

      // Consume individual properties to entangle tracking.
      // https://github.com/emberjs/ember.js/issues/19277
      Object.values(positional);
      Object.values(named);

      return;
    }
    fn(positional.slice(1), named);
```

I think just calling `Object.values(...)` should be sufficient, as this already triggers the getters for each property.

Reply:
Do we need to also access `positional.length` to support `positional.pushObject('new tail')`?

Reply:
Similarly, would this work when a new key is added to `named` via `named.set('newKey', true)`?

Reply:
You don't need to verify these things, if you don't wanna. I'll check this out later today, add tests and migrate the CI setup to GitHub Actions.q

Reply:
I force-pushed the suggested change. However, I'm not sure about the things you're asking. If you can check them later, that would be great, thanks!

Note that this is a "hack" around (I guess) the Ember bug. If they fix it soon, you could then revert that change.

Reply:
hm, `Object.values` doesn't work on an array:

```
Object.keys() was called on the positional arguments array for a helper, which is not supported. This function is a low-level function that should not need to be called for positional argument arrays. You may be attempting to iterate over the array using for...in instead of for...of.
```

Reply:
`Object.values([1, 2, 3])` does work on an array. It's rather that the Ember args proxy apparently has a bug and doesn't support being used with `Object.values(positional)` 🙈

Reply:
@buschtoens - I'm not sure when the bug is going to be fixed on Ember's side - do you mind merging this and releasing a new version and later on you can revert it when Ember fixes it?